tests/e2e/vppagent.go

```diff
@@ -61,9 +83,33 @@ func startAgent(ctx *TestCtx, name string, opts *AgentOpt) (*Agent, error) {
 	if err != nil {
 		return nil, errors.Errorf("can't start agent %s due to: %v", name, err)
 	}
+	client, err := ctl.NewClient(agent.IPAddress())
```

Comment:
No need for temporal `client` variable, just `agent.Client,err = ctl.NewClient(...)`